src/fieldlist_SAMUNICON.jsonc

```diff
@@ -0,0 +1,68 @@
+// All text to the right of an unquoted "//" is a comment and ignored, as well
+// as blank lines (JSONC quasi-standard.) JSON doesn't support line continuation
+// in strings, so you'll want to turn word wrap on in your editor.
+//
+{
+  "convention_name" : "SAMUNICON",
+  "axes" : {
+    "lon" : {"axis" : "X", "MDTF_envvar" : "lon_coord"},
+    "lat" : {"axis" : "Y", "MDTF_envvar" : "lat_coord"},
+    "lev" : {"axis" : "Z", "MDTF_envvar" : "lev_coord"},
+    "time" : {"axis" : "T", "MDTF_envvar" : "time_coord"}
+  },
+  "var_names" : {
+    "hyam_var" : "hyam",
+    "hybm_var" : "hybm",
+    "zg_var" : "Z3",
+    "pr_var" : "PRECT",
+    "rlut_var" : "FLUT",
+    "tas_var" : "TREFHT",
+    "ps_var" : "PS",
+    "qa_var" : "Q",
+    "u_var" : "U",
+    "v_var" : "V",
+    "u200_var" : "U200",
+    "v200_var" : "V200",
+    "u250_var" : "U250",
+    "v250_var" : "V250",
+    "u850_var" : "U850",
+    "v850_var" : "V850",
+    "omega500_var" : "OMEGA500",
+    "wap_var" : "OMEGA",
+    "z250_var" : "Z250",
+    // Variables for Convective Transition Diagnostics module:
+    // ta: 3D temperature, units = K:
+    "ta_var" : "ta",
+    // prw: Column Water Vapor (precipitable water vapor), units = mm (or kg/m^2)
+    "prw_var" : "prw",
+    // tave: Mass-Weighted Column Average Tropospheric Temperature, units = K
+    "tave_var" : "tave",
+    // Vertically-Integrated Saturation Specific Humidity, units = mm (or kg/m^2)
+    "qsat_int_var" : "qsat_int",
+    // Variables for SM_ET_coupling module
+    "mrsos_var" : "mrsos",
+    "evspsbl_var" : "evspsbl",
+    // currently unused
+    "prc_var" : "PRECC",
+    "prls_var" : "PRECL",
+    "FSNTOA_var" : "FSNTOA",
+    "ts_var" : "TS",
+    "LANDFRAC_var" : "LANDFRAC",
+    "tauu_var" : "TAUX",
+    "CLDTOT_var" : "CLDTOT",
+    "ICEFRAC_var" : "ICEFRAC",
+    "psl_var" : "PSL",
+    "siconc_var" : "siconc"
+  },
+  "units" : {
+    // TODO: assign to variables; do honest unit conversion
+    // factor * [model native unit] = flux in kg m-2 s-1
+    "pr_conversion_factor" : 1000,
+    "prc_conversion_factor" : 1000,
+    "prls_conversion_factor" : 1000,
+    // 0 for CMIP/GFDL date index start, 1 for CESM native date index start
+    "date_int_offset" : 1,
+    // pressure axis: factor * [model native unit] = pressure in Pa
+    "pressure_to_Pa_factor" : 100
```

Comment:
As Tom pointed out, this file should be deleted. However, T=these look like environment variables. If the POD requires them,  you should move them to the pod_env_vars section of the POD settings.jsonc file.